composer.json

```diff
@@ -41,7 +41,7 @@
   },
   "config": {
     "platform": {
-      "php": "8.0"
+      "php": "8.1"
```

Comment:
I believe this setting here affects the installed packages on this library only and will not affect the project when we install the package. Is it okay to keep the version `8.0` ?

Reply:
It should be, but I wanted the test suite to run with Carbon V3 as well. I have reverted this change now and only updated the nesbot/carbon line.